_includes/pbc/all/install-features/202404.0/install-the-comments-feature.md

```diff
@@ -0,0 +1,475 @@
+
+| NAME                        | VERSION          | INSTALLATION GUIDE                                                                                                                                                                                                     |
+|-----------------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Customer Account Management | {{page.version}} | [Install the Customer Account Management feature](/docs/pbc/all/customer-relationship-management/{{page.version}}/base-shop/install-and-upgrade/install-features/install-the-customer-account-management-feature.html) |
+| Spryker Core                | {{page.version}} | [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/{{page.version}}/install-and-upgrade/install-features/install-the-spryker-core-feature.html)                                                            |
+
+### 1) Install the required modules
+
+To install the required modules using Composer:
+
+```bash
+composer require spryker-feature/comments: "{{page.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules have been installed:
+
+| MODULE            | EXPECTED DIRECTORY                 |
+|-------------------|------------------------------------|
+| Comment           | vendor/spryker/comment             |
+| CommentDataImport | vendor/spryker/comment-data-import |
+
+{% endinfo_block %}
+
+### 2) Set up configuration
+
+Add the following configuration to your project:
+
+| CONFIGURATION                                                          | SPECIFICATION                                      | NAMESPACE          |
+|------------------------------------------------------------------------|----------------------------------------------------|--------------------|
+| CommentConfig::getAvailableCommentTags()                               | Allows saving comment tags to the database. | Pyz\Shared\Comment |
+| See the regular expression in `config/Shared/config_default.php`. | Blocks access for customers that are not logged in.     | None               |
+
+**src/Pyz/Zed/Comment/CommentConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Shared\Comment;
+
+use Spryker\Shared\Comment\CommentConfig as SprykerCommentConfig;
+
+class CommentConfig extends SprykerCommentConfig
+{
+	/**
+	 * @return list<String>
+	 */
+	public function getAvailableCommentTags(): array
+	{
+		return [
+			'delivery',
+			'important',
+		];
+	}
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Add or remove a comment tag by changing the `getAvailableCommentTags()` method in the `CommentConfig` class.
+2. On the **Cart** page, verify that the comment tag you've added or removed is now available or not available respectively.
```

Comment:
Are there any comment tags before I install this feature? Because, if not, it seems to me that the verification should be `On the Cart page, make sure you can add comment tags`. Correct me if I'm wrong :)

Reply:
When you say "by default" do you mean before I install this feature OR this feature is shipped with these two default comment tags?

Reply:
This feature is shipped with two default tags)